test/index.test.js

```diff
@@ -33,7 +33,7 @@ function execLoader(filename, callback) {
 			return this.callback;
 		}
 	};
-	var res = loader.call(context, fs.readFileSync(filename, "utf-8"));
+	var res = loader.call(context, fs.readFileSync(filename, "utf-8").replace(/\r/g, ''));
```

Comment:
Split in two lines: (e.g. `var res = ... <newline> res = res.replace(/\r/g, '')` and add a comment explaining the addition.